packages/fast-components-styles-msft/src/button/index.ts

```diff
@@ -235,6 +248,11 @@ const styles: ComponentStyles<ButtonClassNameContract, DesignSystem> = {
     },
     button__disabled: {
         ...applyDisabledState(),
+        "& $button_beforeContent, & $button_afterContent": {
+            "@media (-ms-high-contrast:active)": {
+                fill: "white", // ButtonHighlight cannot be passed to -ms-high-contrast but white is safe in this context

```

Comment:
Can you elaborate on this? I see `ButtonHighlight` being passed above to a -ms-high-contrast query. My concern with "white" is that there are multiple high contrast modes supported by Windows and white may not meet contrast in all of them. Can we confirm these worked in all of the modes?

Reply:
The compound JSS statement was causing the issue. Separating `& $button_beforeContent` and `& $button_afterContent` allowed the usage of `ButtonHighlight`.